src/components/NavBar/NavBar.tsx

```diff
@@ -9,8 +11,10 @@ const NavBar = () => {
           Marketplace
         </NavLink>
           {/* later replace with link to wallet connection */}
-          <div>{useWalletConnect()}</div>
-        <NavLink to="/CollectionsPage">View Assets</NavLink>
+        <div>{ReflectWalletState()}</div>
```

Comment:
I think you're calling a React component as a function - in this case I think it's not breaking things because you're not passing any data in
Component are called in this syntax `<ReflectWalletState />` and as you can see the name is probably not the best for a component